apps-rendering/src/components/embedWrapper.tsx

```diff
@@ -356,11 +364,15 @@ const getSourceDetailsForEmbed = (embed: Embed): SourceDetails => {
 			};
 		case EmbedKind.Generic:
 		case EmbedKind.TikTok:
-		case EmbedKind.EmailSignup:
 			return {
 				source: embed.source,
 				sourceDomain: embed.sourceDomain,
 			};
+		case EmbedKind.EmailSignup:
+			return {
+				source: some('Guardian'),
+				sourceDomain: some('www.theguardian.com'),
```

Comment:
Do you think it would be worth storing these two fields in a standalone file and also have `newsletterEmbeds.ts` pull its usage of `www.theguardian.com` from the same variable in that file? An email embeds config file perhaps?

It's not a problem for now, but I'm thinking of a future scenario where we change the way email embeds work (perhaps hosting them on a different domain). Because these lines aren't coupled to the rest of the embed code, whoever makes the change may not realise this needs changing too. If that happens then the embed wrapper will display the wrong information.

The other option is to include these two fields in the main `EmailSignup` type, and parse them from CAPI.

Reply:
I think my preferred option here is to include `source` and `sourceDomain` in the `EmailSignup` type and parse them from CAPI. Although it would be nice to have them in a separate field and use them from `newsletterEmbeds.ts`, we need them in a slightly different format there (as an `origin` rather than just `domain`) - unless we refactor the origin checking too.
What do you think?

Reply:
We decided to parse  `source` and `sourceDomain` from CAPI